Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.11-slim@sha256:4d091e6e8ea62ee443917ffa62106f08da104c133026bcc8f153a1db92fe27cd
+
+# Configuración Poetry (https://python-poetry.org/docs#ci-recommendations)
```

Comment:
Primero, no pongáis comentarios en el código. Esto lo he repetido muchas veces. Comentarios en código es automáticamente deuda técnica, porque no hay forma posible de mantenerlos sincronizados con lo que comentan.
Segundo, *siempre* tienes que ser consciente de la situación y de si las recomendaciones que lees se aplican a tu caso o no. En este caso, ese enlace dice literalmente:

> Unlike development environments, where making use of the latest tools is desirable, in a CI environment reproducibility should be made the priority. Here are some suggestions for installing Poetry in such an environment.

Te lo voy a poner en negrita a ver si lo entiendes mejor

> Unlike *development environments*, where *making use of the latest tools is desirable*, in a CI environment reproducibility should be made the priority. Here are some suggestions for installing Poetry in such an environment.

Un contenedor de test, ¿en qué tipo de entorno se usa? Es más, `poetry`, para lo que nosotros lo usamos, ¿qué tipo de herramienta es?